server/tests/__mocks__/cohortsMock.js

```diff
@@ -0,0 +1,50 @@
+const cohortsMock = [
+  {
+    _id: { $oid: '5ccfe9631ad8a35fee6fbc75' },
+    favIds: [],
+    isArchived: false,
+    memberIds: ['5c9dc5a619ae7924074940f4'],
```

Comment:
shouldn't memberIds be an array of ObjectIds, not strings?

Reply:
doesn't matter. 
Helpers deals both with ObjectIds and string ids.
Tests pass in every case